src/components/section/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import {
+  Box,
+  BoxProps,
+  chakra,
+  RecipeVariantProps,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { sectionRecipe } from 'src/themes/shared/slotRecipes/section';
+
+import { Stack } from '../stack';
+
+type SectionVariantProps = RecipeVariantProps<typeof sectionRecipe>;
+
+export type SectionProps = SectionVariantProps & {
+  variant?: 'hero' | 'regular';
+  title: string;
+  text?: string;
+  image?: ReactNode;
+  maxImgW?: BoxProps['maxWidth'];
+};
+
+export const Section: FC<SectionProps> = ({ maxImgW = '2xl', ...props }) => {
+  // const recipe = useSlotRecipe({ key: 'section' });
+  const recipe = useSlotRecipe({ recipe: sectionRecipe });
```

Comment:
We go by key in other components. The commented-out code should just work™️

```suggestion
  const recipe = useSlotRecipe({ key: 'section' });
```